### ISSUE TITLE
Filtering a model's child inside a form partial view

### ISSUE THREAD

**Issue by ashleywysocki** [activeadmin/activeadmin#2525]
I'm using Rails 3.2.13 and Active Admin 0.5.1 and using a form partial for the form. On the model's form partial page, I have a table of all the model's (has many) children (using a arb partial page with table_for) and I want to filter all the children based on certain criteria. So basically Book has many pages and I want to filter the page's attributes. I've googled everything I can about filters and looked at the source code but I can't figure out how you use filters when not on the index page and for a child element instead of the parent. When I try to use filter on my partial:

```
filter :bookmark, :collection => book.pages
```

I get:

```
undefined method `filter' for #<Arbre::Context:0x007fdab572a078> 
```

Basically I want to use index page of the model's child with the table and filtering in a form partial. Is there any way to do this?
